### PR TITLE
Live k8s test

### PR DESCRIPTION
**Pull request by tanyaveksler**
Implemented live k8s test workflow and related scripts

### CODE REVIEW

File: .github/workflows/test-push-live-k8s-cluster.yml [hunk 1]
```diff
@@ -16,33 +16,25 @@ jobs:
...
+      - name: Install dependencies
+        run: |
+          python -m pip install --upgrade pip
+          pip install flake8 pytest
+          if [ -f requirements.txt ]; then pip install -r requirements.txt; fi
 
       - uses: actions/setup-python@v2.2.2
         with:
           python-version: 3.7
           architecture: x64
```
Comment (zivnevo) [inline .github/workflows/test-push-live-k8s-cluster.yml:34]: Why do we have to run `setup-python` twice?

File: tests/run_live_k8s_tests.py [hunk 2]
```diff
@@ -1,6 +1,5 @@
 import sys
 import os
-from fnmatch import fnmatch
 from time import time
 from ruamel.yaml import YAML
 import argparse
```
Comment (zivnevo) [inline tests/run_live_k8s_tests.py:5]:
```suggestion
import argparse
from ruamel.yaml import YAML
```
standard imports should come before 3rd-party imports

File: tests/run_live_k8s_tests.py [hunk 3]
```diff
@@ -34,6 +33,7 @@ def main(argv=None):
     :return:
     """
     base_dir = os.path.abspath(os.path.dirname(sys.argv[0]))
+    os.chdir(base_dir)
 
     # Deploy namespaces, pods and network policies
```
Comment (zivnevo) [inline tests/run_live_k8s_tests.py:38]: Clean commented-out code

**Review by zivnevo — CHANGES_REQUESTED** [review]
A few small fixes

File: .github/workflows/test-push-live-k8s-cluster.yml (hunk 1 above)
Comment (tanyaveksler) [reply .github/workflows/test-push-live-k8s-cluster.yml]: Removed the second setup-python

File: tests/run_live_k8s_tests.py (hunk 3 above)
Comment (tanyaveksler) [reply tests/run_live_k8s_tests.py]: Done

**Review by zivnevo — APPROVED** [review]
Very cool. Thanks!

**Pull request merged**